src/ReactiveUI.Wpf/ActivationForViewFetcher.cs

```diff
@@ -55,9 +56,50 @@ public IObservable<bool> GetActivationForView(IActivatableView view)
                 x => fe.Unloaded += x,
                 x => fe.Unloaded -= x);
 
+            var windowActivation = GetActivationForWindow(view);
+
+            var dispatcherActivation = GetActivationForDispatcher(fe);
+
             return viewLoaded
                 .Merge(viewUnloaded)
                 .Merge(hitTestVisible)
+                .Merge(windowActivation)
+                .Merge(dispatcherActivation)
+                .DistinctUntilChanged();
+        }
+
+        private static IObservable<bool> GetActivationForWindow(IActivatableView view)
+        {
+            if (view is not Window window)
+            {
+                return Observable<bool>.Empty;
+            }
+
+            var viewClosed = Observable.FromEvent<EventHandler, bool>(
+                eventHandler =>
+                {
+                    void Handler(object? sender, EventArgs e) => eventHandler(false);
+                    return Handler;
+                },
+                x => window.Closed += x,
+                x => window.Closed -= x);
+
+            return viewClosed
+                .DistinctUntilChanged();
```

Comment:
Probably we don't need a `DistinctUntilChanged()` here as we are applying this operator anyway at line 68